lib/transcoder.ex

```diff
@@ -80,12 +91,21 @@ defmodule Membrane.Transcoder do
                 * a boolean,
                 * a function that receives the input stream format and returns a boolean.
                 """
+              ],
+              override_input_stream_format: [
+                spec: Membrane.StreamFormat.t() | nil,
+                default: nil,
+                description: """
+                Allows to override stream format of the input stream.
+                If nil, the input stream format won't be overriden.
+                """
```

Comment:
I would allow to pass there only `%RemoteStream{content_format: Membrane.MPEGAudio}` as it is the only value that makes sense for now and I would raise if it is set to something different

Reply:
How about `assumed_input_stream`